### DISCUSSION TITLE
I wrote a whole new package to supplement this one. I call it Asciidoc-Prose.

### DISCUSSION THREAD

**Discussion by polyglot-jones** [tajmone/ST4-Asciidoctor#44] [Show and tell]
Hey all,

Years ago, I tinkered with submitting some new features to this ST4-Asciidoctor package that were really outside the scope and ended up retracting those PRs. So, instead I began to collect them on the side. Well, I finally got around to creating a whole new package to house them. It's here: https://github.com/gruntwurk/Asciidoc-Prose. Let me know what you think.

Thanks,

// Craig

**Comment by tajmone** [comment]
> Let me know what you think.

I've only managed to read through the presentation of the package so far, but hadn't yet had a chance to actually test it in the editor — if I find some free time in the weekend I'll install it, so I can appreciate it fully. 

This seems a very interesting package for anyone who's into fiction writing (I do, as a hobby).

I'll tell you more once I've tried it out.
